app/notifications/validators.py

```diff
@@ -48,9 +48,9 @@ def check_service_over_daily_message_limit(key_type, service):
             service_stats = services_dao.fetch_todays_total_message_count(service.id)
             redis_store.set(cache_key, service_stats, ex=3600)
 
-        if service.message_limit - int(service_stats) <= 100:
-            current_app.logger.info(f'service {service.id} nearing daily limit'
-                                    f'{service.message_limit} - {service_stats}')
+        if (int(service_stats) / service.message_limit) * 100 > 75:
+            current_app.logger.info(f'service {service.id} daily limit reached '
+                                    f'{int(service_stats) / service.message_limit * 100}%')
```

Comment:
I think it would be more useful to the customer to see how many messages they've sent vs the limit. This isn't when the limit has been reached, that's the next check being made.

Reply:
Nevermind the first part of this, it says in the ticket `Logger message updated to reflect percent`. I think it should be left as "nearing daily limit" though